debian/control

```diff
@@ -0,0 +1,13 @@
+Package: cometvisu
+Replaces: cometvisu-wiregate,cometvisu-0.5,cometvisu-0.8
+Priority: extra
+Maintainer: CometVisu.org <info@cometvisu.org>
+Vcs-Git: https://github.com/CometVisu/CometVisu.git
+Vcs-browser: https://github.com/CometVisu/CometVisu
+Homepage: http://www.cometvisu.org/wiki/CometVisu
```

Comment:
Should the homepage not be the more generic and correct  http://www.cometvisu.org/